media/com_languages/js/admin-override-edit-refresh-searchstring.es6.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @copyright  Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+document.addEventListener('DOMContentLoaded', () => {
+  document.getElementById('jform_searchstring').addEventListener('focus', (event) => {
+    if (!Joomla.overrider.states.refreshed) {
+      const expired = document.getElementById('overrider-spinner').dataset.searchstringexpired;
```

Comment:
Please don't use `dataset.searchstringexpired`, but instead, getAttribute('data-searchstringexpired'). It's much faster

Reply:
Done